core/input/input_map.cpp

```diff
@@ -474,6 +474,7 @@ const OrderedHashMap<String, List<Ref<InputEvent>>> &InputMap::get_builtins() {
 
 	inputs = List<Ref<InputEvent>>();
 	inputs.push_back(InputEventKey::create_reference(KEY_TAB));
+	inputs.push_back(InputEventKey::create_reference(KEY_ENTER));
```

Comment:
I'd add a comment for future readers to make sure they understand that this is intentional:

```suggestion
	// Allow both Tab and Enter. This matches the default behavior found in most code editors.
	inputs.push_back(InputEventKey::create_reference(KEY_TAB));
	inputs.push_back(InputEventKey::create_reference(KEY_ENTER));
```

Reply:
Why would anyone think otherwise though? Many other shortcuts have alternate mappings and they don't have any comment either.

Reply:
In general, I think it's important to document design decisions as much as possible. Code itself doesn't provide a way to do so, so you need to sprinkle comments here and there.

Reply:
I still don't understand how is it different from e.g. `ui_redo`, which has Ctrl+Shift+Z and Ctrl+Y bound. Also this is how it used to work in 3.x, the issue is even labelled as regression. So I don't see why this one particular shortcut needs further explanation.

Reply:
I think it's OK as is, but I do also think it's worth adding comments to explain this kind of thing when it's not obvious. The Ctrl+Shift+Z and Ctrl+Y one is definitely one which could use a comment IMO, moreso than Tab and Enter for accepting a completion dialog.